lib/salus/repo.rb

```diff
@@ -95,5 +95,18 @@ class Repo
     def initialize(path_to_repo = nil)
       @path_to_repo = path_to_repo
     end
+
+    ##
+    # @param [Array] args CLI rg command and options to run
+    # @return [Array<String>] Relative path of files returned
+    # from running the command.
+    def run_rg(*args)
+      data = IO.popen(args, chdir: @path_to_repo).read
+      return [] if data == ""
```

Comment:
When would data be `""`?
Is it when a directory is empty?  

What about when data is `nil`?